src/applications/gqrx/remote_control_settings.ui

```diff
@@ -48,7 +48,6 @@
         <number>65535</number>
        </property>
        <property name="value">
-        <number>7356</number>
```

Comment:
Did you make this change in Qt Creator / Designer or did you just remove the line in the .ui file?